### ISSUE TITLE
Footer covers login fields on mobile when keyboard launches

### ISSUE THREAD

**Issue by tlofgren** [SnapChallenge/SnapChallenge#3]

**Comment by shaheenp** [comment]
Looks like you pushed some HTML/CSS changes around the same time you created this issue. Is it solved? It's not happening on my iPhone.

**Comment by tlofgren** [comment]
Is the footer still visible on your phone once the keyboard launches, on either signup or login? It is on mine because the page viewport is just shrinked when the keyboard is visible. Which is the source of the problem. No idea how to fix it, so I created the issue.

**Comment by shaheenp** [comment]
Nope, footer does not show up on my phone on signup and login pages when the keyboard is open. What phone do you have?

**Comment by tlofgren** [comment]
LG G3. Jacob has the same phone, I'll ask if this happens to him.
